android/java/org/chromium/chrome/browser/crypto_wallet/adapters/CryptoFragmentPageAdapter.java

```diff
@@ -25,29 +24,36 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class CryptoFragmentPageAdapter extends FragmentStatePagerAdapter {
-    private static final int PORTFOLIO_FRAGMENT_POSITION = 0;
-    private static final int NFT_GRID_FRAGMENT_POSITION = 1;
-    private static final int TRANSACTIONS_ACTIVITY_FRAGMENT_POSITION = 2;
-    private static final int ACCOUNTS_FRAGMENT_POSITION = 3;
-    private static final int MARKET_FRAGMENT_POSITION = 4;
+/**
+ * Fragment adapter for the main sections of Brave Wallet displayed by {@link BraveWalletActivity}.
+ */
+public class CryptoFragmentPageAdapter extends FragmentStateAdapter {
+    public static final int PORTFOLIO_FRAGMENT_POSITION = 0;
+    public static final int NFT_GRID_FRAGMENT_POSITION = 1;
+    public static final int TRANSACTIONS_ACTIVITY_FRAGMENT_POSITION = 2;
+    public static final int ACCOUNTS_FRAGMENT_POSITION = 3;
+    public static final int MARKET_FRAGMENT_POSITION = 4;
 
     private final List<String> mTitles;
 
     private PortfolioFragment mCurrentPortfolioFragment;
 
-    public CryptoFragmentPageAdapter(FragmentManager fm, Context context) {
-        super(fm, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT);
-        Resources resources = context.getResources();
+    public CryptoFragmentPageAdapter(FragmentActivity fm) {
+        super(fm);
+        Resources resources = fm.getResources();
```

Comment:
nit: `fm` should be `activity`, no longer a `FragmentManager`

Reply:
👍